io_scene_niftools/operators/common_op.py

```diff
@@ -84,24 +84,15 @@ class CommonDevOperator:
         options={'HIDDEN'})
 
 
-class CommonImportScale:
+class CommonScale:
     # Number of nif units per blender unit.
-    scale_correction_import: bpy.props.FloatProperty(
-        name="Scale Correction Import",
+    scale_correction: bpy.props.FloatProperty(
+        name="Scale Correction",
         description="Changes size of mesh to fit onto Blender's default grid.",
         default=0.1,
         min=0.001, max=100.0, precision=2)
 
 
-class CommonExportScale:
-    # Number of blender units per nif unit.
-    scale_correction_export: bpy.props.FloatProperty(
-        name="Scale Correction Export",
-        description="Changes size of mesh from Blender default to nif default.",
-        default=10.0,
```

Comment:
I think this is still need for the read part.
default should read the scene level value, otherwise the transform section is not populating
![image](https://user-images.githubusercontent.com/525919/101226207-9f8b6980-368b-11eb-86ee-7273527a6257.png)

Reply:
Scale does appear under transform for import.
We can also include it for export if you have a getter to retrieve the default value from the current scene. I wasn't sure how to code that.